components/script/dom/svgsvgelement.rs

```diff
@@ -224,7 +193,13 @@ impl VirtualMethods for SVGSVGElement {
         let htmlString = self.upcast::<Element>().GetOuterHTML();
         match htmlString {
             Ok(domString) => {
-                *self.htmlString.borrow_mut() = Some(domString.to_string());
+                let svg_string = domString.to_string();
+                let cloned_svg_string = svg_string.clone();
+                let webgl_sender = &self.webgl_sender;
+                *self.html_string.borrow_mut() = Some(svg_string);
+                if let Some(webgl_sender) = webgl_sender {
+                    webgl_sender.send_rebuild_svg(cloned_svg_string).unwrap();
+                }
```

Comment:
Add this as well, to be sure that layout knows that it needs to redraw the canvas:
```
self.upcast::<Node>().dirty(NodeDamage::OtherNodeDamage);
```